### PR TITLE
Promote bluez, libicall, and libell to base repo

### PR DESCRIPTION
**Pull request by dallasd1**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Move packages bluez, libical, and libell from SPECS-EXTENDED to SPECS folder.

###### Change Log  

- Moved directories to the SPECS folder
- Bump release number

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Test Methodology

- Local build

### CODE REVIEW

File: SPECS/bluez/bluez.spec [hunk 1]
```diff
@@ -2,7 +2,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Name:    bluez
 Version: 5.63
-Release: 1%{?dist}
+Release: 2%{?dist}
```
Comment (oliviacrain) [inline SPECS/bluez/bluez.spec:5]: Release bumps should be accompanied with an entry in the `%changelog` section

File: SPECS/libell/libell.spec [hunk 2]
```diff
@@ -2,7 +2,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Name:           libell
 Version:        0.48
-Release:        1%{?dist}
+Release:        2%{?dist}
```
Comment (oliviacrain) [inline SPECS/libell/libell.spec:5]: Release bumps should be accompanied with an entry in the `%changelog` section

File: SPECS/libical/libical.spec [hunk 3]
```diff
@@ -4,7 +4,7 @@ Distribution:   Mariner
 Summary:	Reference implementation of the iCalendar data type and serialization format
 Name:		libical
 Version:	3.0.9
-Release:	4%{?dist}
+Release:	5%{?dist}
```
Comment (oliviacrain) [inline SPECS/libical/libical.spec:7]: Release bumps should be accompanied with an entry in the `%changelog` section

**Review by oliviacrain — CHANGES_REQUESTED** [review]
Stealing the standard promotion review spiel from @PawelWMS:

Specs from SPECS-EXTENDED rarely can be simply moved into SPECS without additional work as they have a lower bar in terms of test, spec structure, linting, etc.

Please make sure to now:

- Lint the spec.
- Make sure the ptests (inside the %check section) run successfully.
- Remove all references to other distros (if any). In most cases we tend to keep the parts of the spec, which were enabled for Fedora and remove the other ones.
- Remove the GPG keyring and signatures - we rely on the signatures.json files.
- Make sure all automated PR checks pass (hard to miss, you probably won't be able to merge before the required ones are resolved).

File: SPECS/libical/libical.spec (hunk 3 above)
Comment (dallasd1) [reply SPECS/libical/libical.spec:7]: changelog updated

File: SPECS/libell/libell.spec (hunk 2 above)
Comment (dallasd1) [reply SPECS/libell/libell.spec:5]: changelog updated

File: SPECS/bluez/bluez.spec (hunk 1 above)
Comment (dallasd1) [reply SPECS/bluez/bluez.spec:5]: changelog updated

**Comment by PawelWMS** [comment]
> Stealing the standard promotion review spiel from @PawelWMS Pawel Winogrodzki FTE:
> 
> Specs from SPECS-EXTENDED rarely can be simply moved into SPECS without additional work as they have a lower bar in terms of test, spec structure, linting, etc.
> 
> Please make sure to now:
> 
> * Lint the spec.
> * Make sure the ptests (inside the %check section) run successfully.
> * Remove all references to other distros (if any). In most cases we tend to keep the parts of the spec, which were enabled for Fedora and remove the other ones.
> * Remove the GPG keyring and signatures - we rely on the signatures.json files.
> * Make sure all automated PR checks pass (hard to miss, you probably won't be able to merge before the required ones are resolved).

One more item, which came to my mind right now: make sure the packages have all of their run-time dependencies satisfied.

**Comment by PawelWMS** [comment]
Please re-base the change against the `main` branch - we don't merge directly into `2.0` except during our release process.

**Review by PawelWMS — CHANGES_REQUESTED** [review]
+1 on Olivia's comments and please re-base the PR against `main` instead of `2.0`.

**Comment by dallasd1** [comment]
Rebasing these changes with a PR based on main instead of 2.0

**Pull request closed without merging**